UPGRADING.md

```diff
@@ -51,6 +51,48 @@ class TwitterAPI < Grape::API
 end
 ```
 
+#### Recognizing Path
+
+Due to the changes done in [#2379](https://github.com/ruby-grape/grape/pull/2379), Grape now takes in mind the types of the configured `route_params` in order to determine the endpoint that matches with the performed request. 
+
+So taking into account this `Grape::API` class
+
+```ruby
+class Books < Grape::API
+  resource :books do
+    route_param :id, type: Integer do
+      # GET /books/:id
+      get do
+        #...
+      end
+    end
+
+    resource :share do
+      # POST /books/share
+      post do
+      # ....
+      end
+    end
+  end
+end
+```
+
+This was the behavior before the changes:
```

Comment:
Follow the convention, say "Before:" and "After:"

Reply:
Totally right. I've just applied it! 👏